Source/JavaScriptCore/wasm/WasmLLIntGenerator.cpp

```diff
@@ -118,11 +136,17 @@ class LLIntGenerator : public BytecodeGeneratorBase<GeneratorTraits> {
             return ControlType(signature, stackSize - signature->argumentCount(), WTFMove(continuation), ControlTry { WTFMove(tryLabel), tryDepth });
         }
 
+        static ControlType createTryTable(BlockSignature signature, unsigned stackSize, Ref<Label>&& tryLabel, RefPtr<Label>&& continuation, unsigned tryDepth, Vector<ControlTryTable::TryTableTarget, 8> targets)
```

Comment:
Take `Vector<ControlTryTable::TryTableTarget, 8>` as `Vector<ControlTryTable::TryTableTarget, 8>&&` and use `WTFMove(targets)`